.github/workflows/pr-check.yaml

```diff
@@ -13,12 +13,9 @@ jobs:
         with:
           node-version: '20'
       - name: Install AsyncAPI Generator
-        # The @asyncapi/generator version is pinned to the latest version that's
-        # compatible with the current latest version of @asyncapi/html-template
-        # (v2.3.5).
-        run: npm install -g @asyncapi/generator@1.17.25
+        run: npm install -g @asyncapi/cli
       - name: Generate Organization AsyncAPI UI
-        run: ag docs/organizations/asyncapi.external.organizations.yaml @asyncapi/html-template -o docs/organizations/asyncapi.external.organizations
+        run: asyncapi generate fromTemplate docs/organizations/asyncapi.external.organizations.yaml @asyncapi/html-template@2.3.14 -o docs/organizations/asyncapi.external.organizations
```

Comment:
With the latest version of the AsyncAPI generator, the `ag` command [is deprecated](https://github.com/asyncapi/generator/releases/tag/%40asyncapi%2Fgenerator%402.6.0).

And according to the AsyncAPI HTML Template docs, we need to pin the HTML template version to `2.3.14` since this is using template version v1 or v2 with AsyncAPI Generator v1.